js/sticky-header.js

```diff
@@ -0,0 +1,48 @@
+$(function() {
+    // header hide and show on scroll up/down
+    var didScroll;
+    var lastScrollTop = 0;
+    var delta = 5;
+    var initialHeaderHeight = $('header.primary').outerHeight();
+
+    // set top padding on body so header doesn't overlap content
+    $('body').css('padding-top', initialHeaderHeight);
+
+    $(window).scroll(function(event){
+        didScroll = true;
+    });
+
+    setInterval(function() {
+        if (didScroll) {
+            hasScrolled();
+            didScroll = false;
+        }
+    }, 250);
+
+    var hasScrolled = function() {
+        var headerHeight = $('header.primary').outerHeight();
```

Comment:
You should cache your selectors 

`var $headerPrimary = $('header.primary);
 var headerHeight =  $headerPrimary.outerHeight();
`